### DISCUSSION TITLE
WS2812B Rainbow effect not working

### DISCUSSION THREAD

**Discussion by Meghpakhi** [arendst/Tasmota#19023] [Q&A]
WS2812b rainbow effect not working on tasmota. Running only 1 time after put 11 scheme. Then the white light turns on. Scheme 4 working but scheme 11, 12, 13 not working. please help

**Comment by sfromis** [comment]
Do check the log (probably with log level 3) to see what's happening.

**Reply by Meghpakhi** [reply]
![scheme 11](https://github.com/arendst/Tasmota/assets/138502793/36bfdc84-a476-44bf-9eaf-d4554540d112)
on console CMD Scheme 11. no error
